apps/documentation/src/routes/documentation/routing/api-routes.mdx

```diff
@@ -0,0 +1,173 @@
+import MetaTags from '@/components/MetaTags'
+
+<MetaTags
+  title="Tuono - API routes"
+  canonical="https://tuono.dev/documentation/routing/api-routes"
+/>
+
+import Breadcrumbs from '@/components/Breadcrumbs'
+
+<Breadcrumbs breadcrumbs={[{ label: 'API Routes' }]} />
+
+# API Routes
+
+Just like with [NextJS API routes ](https://nextjs.org/docs/pages/building-your-application/routing/api-routes). Tuono API routes provide a solution to build a public API.
```

Comment:
We shouldn't reference Next.js

```suggestion
API routes provide a solution to build a public API with Tuono ⚡.

Any file inside the folder `src/routes/api` is mapped to `/api/*` path and will be treated as an API endpoint instead of a React route.
```